native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -33,27 +33,42 @@ const underlayColor = Platform.select({
 // These should probably be their own tokens.
 // For now they are derived from existing tokens.
 const largeHeight = tokens.spacingSm * 12;
-const buttonBorderWidth = tokens.borderSizeLg;
+const buttonBorderWidth = Platform.OS === 'android' ? tokens.borderSizeSm : tokens.borderSizeLg;
```

Comment:
Maybe this could be implemented in the tokens themselves, by adding aliases for `buttonBorderWidth` to the platform-specific tokens?

Reply:
needs to be refactored for both iOS and Android, so I'll do that in a separate PR as it goes beyond the scope of this one